src/pages/Content/index.ts

```diff
@@ -1,33 +1,51 @@
 import select from "select-dom";
 import getCommitElements from "@pages/Content/getCommitElements";
 import createCommitsCollapse from "@pages/Content/createCommitsCollapse";
+import removeAlreadyCreatedDetail from "@pages/Content/removeAlreadyCreatedDetail";
+import { DETAIL_CLASS_NAME } from "@pages/Content/constansts";
 
-const detect = (targetElement: HTMLElement) => {
-  const observer = new MutationObserver(function (mutations) {
-    mutations.forEach(() => {
-      const finded = select("markdown-toolbar", targetElement);
+const findAllToolbar = (): HTMLElement[] => {
+  const toolbars = select.all("markdown-toolbar");
+  return toolbars.filter(Boolean);
```

Comment:
변수 인라인 처리해주면 어떨까요? 
```suggestion
const findAllToolbar = (): HTMLElement[] => {
 return select.all("markdown-toolbar").filter(Boolean);
```

코드를 다시 보니 select인자로 넣는 값이 어떤 건지 알기가 어려워서 주석을 간단히 적어봐야겠네요

Reply:
반영해버리기